apps/openchallenges/organization-service/src/test/java/org/sagebionetworks/openchallenges/organization/service/model/mapper/OrganizationMapperTest.java

```diff
@@ -0,0 +1,111 @@
+package org.sagebionetworks.openchallenges.organization.service.model.mapper;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.sagebionetworks.openchallenges.organization.service.model.dto.OrganizationDto;
+import org.sagebionetworks.openchallenges.organization.service.model.entity.OrganizationEntity;
+
+public class OrganizationMapperTest {
+  @Test
+  public void ConvertToEntity_ShouldReturnDtoProperties_WhenDtoPropertiesPassed() {
+
+    // create new Dto with properties to be copied to organization entity
+    OrganizationDto organizationDto = new OrganizationDto();
+    organizationDto.setName("Test Organization");
+    organizationDto.setDescription("This is a test organization");
+    organizationDto.setId(4599L);
+    organizationDto.setEmail("testemail@test.edu");
+    organizationDto.setLogin("login");
+    organizationDto.setAvatarKey("avatar Key");
+    organizationDto.setWebsiteUrl("url");
+    organizationDto.setChallengeCount(7);
+    organizationDto.setAcronym(null);
+
+    OrganizationMapper mapper = new OrganizationMapper();
+
+    OrganizationEntity entity = mapper.convertToEntity(organizationDto);
+
+    Long retrievedId = entity.getId();
+    String retrievedName = entity.getName();
+    String retrievedEmail = entity.getEmail();
+    String retrievedLogin = entity.getLogin();
+    String retrievedAvatarKey = entity.getAvatarKey();
+    String retrievedWebsiteUrl = entity.getWebsiteUrl();
+    Integer retrievedChallengeCount = entity.getChallengeCount();
+    String retrievedDescription = entity.getDescription();
+    String retrievedAcronym = entity.getAcronym();
+
+    Long retrievedId2 = organizationDto.getId();
+    String retrievedName2 = organizationDto.getName();
+    String retrievedEmail2 = organizationDto.getEmail();
+    String retrievedLogin2 = organizationDto.getLogin();
+    String retrievedAvatarKey2 = organizationDto.getAvatarKey();
+    String retrievedWebsiteUrl2 = organizationDto.getWebsiteUrl();
+    Integer retrievedChallengeCount2 = organizationDto.getChallengeCount();
+    String retrievedDescription2 = organizationDto.getDescription();
+    String retrievedAcronym2 = organizationDto.getAcronym();
```

Comment:
See my comment about "actual" Vs. "expected" in the other open PR. Here you would rename "retrieved" by "actual".

Reply:
Got it thanks! Updating it now.